forms-flow-theme/scss/inputBox.scss

```diff
@@ -49,3 +49,63 @@ $vadiation-astrisk-color: var(--ff-base-600, blue);
 .vadiation-astrisk {
   color: $vadiation-astrisk-color;
 }
+
+.input-dropdown{
+  position: relative; 
+  
+  .border-input{  
+    border-radius:var(--radius-0100) var(--radius-0100) 0px 0px !important;
+  }
+
+  .input-group-text svg {
+    transition: transform 0.3s ease; 
+  }
+  
+  .collapsed + .input-group-text svg {
+    transform: rotate(180deg);
+  }
+  .list-group{
+    position: absolute; 
+    top: 100%; 
+    left: 0;
+    right: 0;
+    z-index: 100; 
+    display: flex;
+    padding: var(--spacer-050) 0px;
+    flex-direction: column;
+    align-self: stretch;
+    border: 1px solid var(--ff-base-600, #253DF4); 
+    border-top: none;
+    border-radius: 0px 0px var(--radius-0100, 16px) var(--radius-0100, 16px);
+    max-height: 12.5rem;
+    overflow-y: scroll;
+    background-color: var(--ff-white);
+  
+  .list-group-item{
+    &.list-first-item-btn{
+      color: var(--ff-base-600, #253DF4);
+      font-size: var(--font-size-sm);
```

Comment:
these 2 lines can be removed as it will be applied from .list-group-item style

Reply:
Updated.